pandas/tests/base/test_ops.py

```diff
@@ -594,108 +594,6 @@ def test_factorize_repeated(self):
                 expected = o[5:10].append(o[:5])
                 tm.assert_index_equal(uniques, expected, check_names=False)
 
-    def test_duplicated_drop_duplicates_index(self):
-        # GH 4060
-        for original in self.objs:
-            if isinstance(original, Index):
-
-                # special case
-                if original.is_boolean():
-                    result = original.drop_duplicates()
-                    expected = Index([False, True], name="a")
-                    tm.assert_index_equal(result, expected)
-                    continue
-
-                # original doesn't have duplicates
-                expected = np.array([False] * len(original), dtype=bool)
-                duplicated = original.duplicated()
-                tm.assert_numpy_array_equal(duplicated, expected)
-                assert duplicated.dtype == bool
-                result = original.drop_duplicates()
-                tm.assert_index_equal(result, original)
-                assert result is not original
```

Comment:
Now tested in `test_drop_duplicates_no_duplicates` in `tests/indexes/test_common.py`, see below